recipes/freexl/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import glob
+import os
+
+from conans import ConanFile, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment, tools
+
+class FreexlConan(ConanFile):
+    name = "freexl"
+    description = "FreeXL is an open source library to extract valid data " \
+                  "from within an Excel (.xls) spreadsheet."
+    license = ["MPL-1.0", "GPL-2.0-only", "LGPL-2.1-only"]
+    topics = ("conan", "freexl", "excel", "xls")
+    homepage = "https://www.gaia-gis.it/fossil/freexl/index"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "patches/**"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _autotools= None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
```

Comment:
```suggestion
    def configure(self):
        if self.options.shared:
            del self.options.fPIC
```

After a conversation with Conan team, it seems be the correct way. The problem is affecting the package ID with fPIC, when it's invalid.